bundle/manifests/crunchy-bridge-operator.clusterserviceversion.yaml

```diff
@@ -257,7 +276,12 @@ spec:
                 - --leader-elect
                 command:
                 - /manager
-                image: quay.io/example.org/crunchy-bridge-operator:latest
+                env:
+                - name: INSTALL_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                image: quay.io/ahameed/crunchy-bridge-operator:latest
```

Comment:
Let's use our [team Org](https://quay.io/organization/ecosystem-appeng) instead of personal quay here, please.

Reply:
revert to actual, image: quay.io/example.org/crunchy-bridge-operator:latest.  as discussed, will replace this once the operator images are released officially.